airflow/dags/rxclass/rxclass.py

```diff
@@ -21,7 +21,7 @@ def process_concept(class_base_url, concept):
         return []
 
 
-base_url = "https://rxnav.nlm.nih.gov/REST/allconcepts.json?tty=SBD+SCD+GPCK+BPCK"
+base_url = "https://rxnav.nlm.nih.gov/REST/allconcepts.json?tty=IN+PIN+MIN+SCDC+SCDF+SCDFP+SCDG+SCDGP+SCD+GPCK+BN+SBDC+SBDF+SBDFP+SBDG+SBD+BPCK"
```

Comment:
Woof.  How many API calls do all these TTYs generate? 😨

Reply:
When I ran it today, it processed 122,690 concepts......took 13 minutes to run